panel/command/serve.py

```diff
@@ -370,6 +370,21 @@ def customize_kwargs(self, args, server_kwargs):
             module.__dict__['__file__'] = fullpath(args.setup)
             state._setup_module = module
 
+            def setup_file():
+                setup_path = state._setup_module.__dict__['__file__']
+                with open(setup_path) as f:
+                    setup_source = f.read()
+                nodes = ast.parse(setup_source, os.fspath(setup_path))
+                code = compile(nodes, filename=setup_path, mode='exec', dont_inherit=True)
+                exec(code, state._setup_module.__dict__)
+
+            if args.num_procs > 1:
```

Comment:
I need to double-check if this is also a number on Windows.

Reply:
It is.

https://github.com/bokeh/bokeh/blob/b1e7d94e13f11727de1e114e708740eeb19abe41/src/bokeh/command/subcommands/serve.py#L730-L737